### PR TITLE
Bump module dependency to support riscv64 arch

### PR DESCRIPTION
**Pull request by carlosedp**
To be able to build on riscv64, github.com/prometheus/procfs requires a
bump.

Signed-off-by: Carlos de Paula <me@carlosedp.com>

### CODE REVIEW

**Comment by rootfs** [comment]
@carlosedp thanks, can you just upgrade those in `github.com/prometheus/procfs`? we have some patches against `k8s.io` for now

**Comment by carlosedp** [comment]
I've just updated the procfs package, the other bumped modules are pre-reqs from it.

File: vendor/k8s.io/client-go/pkg/version/base.go [hunk 1]
```diff
@@ -1,10 +1,63 @@
...
+// in-tree values are dummy values used for "git archive", which also
+// works for GitHub tar downloads.
+//
+// When releasing a new Kubernetes version, this file is updated by
+// build/mark_new_version.sh to reflect the new version, and then a
+// git annotated tag (using format vX.Y where X == Major version and Y
+// == Minor version) is created to point to the commit that updates
+// pkg/version/base.go
 var (
-    gitMajor = "1"
```
Comment (rootfs) [inline vendor/k8s.io/client-go/pkg/version/base.go:4]: please keep this for now
`git checkout  vendor/k8s.io/client-go/pkg/version/base.go`

File: vendor/k8s.io/component-base/version/base.go [hunk 2]
```diff
@@ -1,10 +1,63 @@
...
+// in-tree values are dummy values used for "git archive", which also
+// works for GitHub tar downloads.
+//
+// When releasing a new Kubernetes version, this file is updated by
+// build/mark_new_version.sh to reflect the new version, and then a
+// git annotated tag (using format vX.Y where X == Major version and Y
+// == Minor version) is created to point to the commit that updates
+// component-base/version/base.go
 var (
-    gitMajor = "1"
```
Comment (rootfs) [inline vendor/k8s.io/component-base/version/base.go:4]: please keep this for now
`git checkout vendor/k8s.io/component-base/version/base.go `

File: vendor/k8s.io/client-go/pkg/version/base.go (hunk 1 above)
Comment (carlosedp) [reply vendor/k8s.io/client-go/pkg/version/base.go]: Done!

**Comment by rootfs** [comment]
thanks for the contribution!

**Pull request merged**